pipeline/extract.py

```diff
@@ -0,0 +1,37 @@
+"""This script extracts story information from the Hacker News API."""
+import pandas as pd
+from requests import get
+
+
+BASE_URL = "https://hacker-news.firebaseio.com/v0/"
+
+
+def get_top_stories() -> list:
```

Comment:
Could tweak this function so that the number of stories returned is an input parameter.

This means that if we want to use a different number later on, we don't need to change any code.

Reply:
Will do, really good idea